scripts/zephyr_module.py

```diff
@@ -95,10 +95,43 @@
     type: seq
     sequence:
       - type: str
+  blobs:
+    required: false
+    type: seq
+    sequence:
+      - type: map
+        mapping:
+          path:
+            required: true
+            type: str
+          sha256:
+            required: true
+            type: str
+          type:
+            required: true
+            type: str
+            enum: ['img', 'lib']
+          version:
+            required: true
+            type: str
+          license-path:
+            required: true
+            type: str
+          url:
+            required: true
```

Comment:
I guess this will become `required: false` when another fetcher is implemented?

Reply:
No, all fetchers will use URLs. And the URL itself is required to identify the fetcher.